src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] machine) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        truck.doWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
```

Comment:
The order of calling `doWork` and `stopWork` seems inconsistent. According to the common practice, a machine should start working before it stops. Please ensure that `doWork` is called before `stopWork` for each machine.